extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -1961,8 +2000,22 @@ describe('amp-a4a', () => {
       sandbox.restore();
     });
 
-    it('should emit upgradeDelay lifecycle ping', () => {
+    it('should emit upgradeDelay lifecycle ping with legacy verifier', () => {
```

Comment:
Should the description here read "...with new verifier" and the test below it "...with legacy verifier"?